config/config.schema.json

```diff
@@ -15,6 +15,19 @@
         "roleName"
       ]
     },
+    "tracing": {
+      "type": "object",
+      "properties": {
+        "address": {
```

Comment:
(bike-shedding) should it be rather `url` to make it explicit that we expect a fully-defined address?